CPRynner/CPRynner.py

```diff
@@ -230,7 +208,7 @@ def _get_username_and_password():
 
         return [username, password]
     else:
-        return [None, None]
+        return [None,None]
```

Comment:
Space after comma is the accepted style in Python (except before a newline, but that is not relevant here).